datalab/datalab.sh

```diff
@@ -111,20 +111,18 @@ function configure_master() {
   mkdir -p datalab-pyspark
   pushd datalab-pyspark
   cp /etc/apt/trusted.gpg .
-  if [[ -f /etc/apt/sources.list.d/backports.list ]]; then
-    cp /etc/apt/sources.list.d/backports.list .
-    ADD_BACKPORTS='ADD backports.list /etc/apt/sources.list.d/'
-  else
-    ADD_BACKPORTS=''
-  fi
   cp /etc/apt/sources.list.d/dataproc.list .
   cat <<EOF >Dockerfile
 FROM ${DOCKER_IMAGE}
-${ADD_BACKPORTS}
+
 ADD dataproc.list /etc/apt/sources.list.d/
 ADD trusted.gpg /tmp/vm_trusted.gpg
-
 RUN apt-key add /tmp/vm_trusted.gpg
+
+RUN apt-get update
```

Comment:
Can we add if check to only apply this change to Debian 10? otherwise there is a risk to break users of other OSs.

Reply:
We have tests for all the OSes and they are passing, so we shouldn't break existing users.